test/redis.js

```diff
@@ -1,12 +1,11 @@
 const redis = require('../lib/redis')
 const {expect} = require('chai')
 
-beforeEach(async function () {
-  if (!redis) return this.skip()
```

Comment:
This line causes all the tests to skip if redis isn't available as it's not nested within a top level describe.